documentation/source/user_section/tutorials/template-registration/customizing-the-algorithm.rst

```diff
@@ -0,0 +1,76 @@
+.. _customizing-registration-section:
+
+Customizing the registration command
+####################################
+
+This page provides recommendations for how to adjust the registration commands if the default parameters are insufficient for your specific data and pipeline.
+
+The ``-param`` argument
+***********************
+
+The ``-param`` argument defines the transformations for each step of the registration process. The easiest way to use ``-param`` is to start with the default values, and adjust one parameter at a time.
+
+.. code-block::
+
+   # Default values for '-param'. (Note: Normally, this would be written as one line, but a line break was added for readability.)
+   -param step=1,type=imseg,algo=centermassrot,metric=MeanSquares,iter=10,smooth=0,gradStep=0.5,slicewise=0,smoothWarpXY=2,pca_eigenratio_th=1.6:
+          step=2,type=seg,algo=bsplinesyn,metric=MeanSquares,iter=3,smooth=1,gradStep=0.5,slicewise=0,smoothWarpXY=2,pca_eigenratio_th=1.6
+
+Step 1 and step 2 can be modified, and additional steps can be added. In general, we recommend that you start with coarse-adjustment steps, then apply finer adjustments with each successive step.
+
+.. note:: Because choosing the right configuration for your data can be overwhelming, feel free to visit the `SCT forum <https://forum.spinalcordmri.org/c/sct/>`_ where you can ask for clarification and guidance.
+
+Common adjustments to ``-param``
+================================
+
+* ``type`` controls the type of data used for registration:
+
+   * ``type=im``: Use the input anatomical image.
+   * ``type=seg``: Use a separate segmentation image. Choose this if your image data contains distortions or artifacts, but you are confident in your segmentation. (For example, if you have manually corrected the segmentation.)
+   * ``type=imseg``: Use both the input image and a segmentation. Only for use with ``algo=centermassrot``.
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/registration_to_template/sct_register_to_template-param-algo.png
```

Comment:
the figure is quite useful for people, and it appears to small here. I would suggest having it appear below the "algo" section and make it bigger. Also, you can crop the figure and move the text legend ([1] ANTs link, etc.) directly in the figure legend.